frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts

```diff
@@ -0,0 +1,67 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { NooBaaObjectBucketClaimModel } from 'packages/noobaa-storage-plugin/src/models';
+import {
+  storageClassDropDown,
+  defaultSC,
+  scDropdown,
+  createBtn,
+} from '../views/bucketClaimCreation.view';
+import {
+  obcCount,
+  clusterHealth,
+  efficiencyValue,
+  savingsValue,
+} from '../views/noobaaDashboardPage.view';
+
+const BROWSER_TIMEOUT = 30000;
+
+describe('Test Object Service Dashboard Page', () => {
+  it('Create an OBC and test equality', async () => {
+    await browser.get(`${appHost}/dashboards/object-service`);
+    await browser.wait(until.visibilityOf(obcCount));
+    await browser.sleep(5000);
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    // Go to create form page
+    await browser.get(
+      `${appHost}/k8s/ns/${testName}/${referenceForModel(NooBaaObjectBucketClaimModel)}/~new/form`,
+    );
+    await browser.wait(until.visibilityOf(scDropdown));
+    await storageClassDropDown.click();
+    await browser.wait(until.visibilityOf(defaultSC), BROWSER_TIMEOUT);
+    await defaultSC.click();
+    await createBtn.click();
+
+    await browser.get(`${appHost}/dashboards/object-service`);
+    await browser.wait(until.visibilityOf(obcCount));
+    await browser.sleep(5000);
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+  });
+
+  it('Checks whether MCG is healthy', async () => {
+    await browser.get(`${appHost}/dashboards/object-service`);
+    expect(clusterHealth.isPresent()).toBeFalsy();
+  });
+
+  it('Checks for correct Efficiency Ratio', async () => {
+    await browser.get(`${appHost}/dashboards/object-service`);
+    await browser.wait(until.visibilityOf(obcCount));
```

Comment:
Why wait for the visibility of obcCount to validate Efficiency Ratio?

Reply:
ack